### PR TITLE
Resolves #2448

### LINKED ISSUE #2448: Page is incomplete
This is missing descriptions 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 7ecea8c5-f14d-4a47-4d80-cce480e4e4ce
* Version Independent ID: 4ea5ce37-1ecb-ade3-3a69-ce2fdcc0513c
* Content: [Provisioning Module](https://docs.microsoft.com/en-us/powershell/module/provisioning/?view=windowsserver2019-ps)
* Content Source: [docset/winserver2019-ps/provisioning/Provisioning.md](https://github.com/MicrosoftDocs/windows-powershell-docs/blob/master/docset/winserver2019-ps/provisioning/Provisioning.md)
* Product: **w10**
* Technology: **powershell-windows**
* GitHub Login: @AndreaBarr
* Microsoft Alias: **v-anbarr**

### PR DESCRIPTION
**Pull request by velkovb**
PR Review [comment](https://github.com/MicrosoftDocs/windows-powershell-docs/pull/2493#issuecomment-869815393)

Add description
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/2448

### CODE REVIEW

File: docset/winserver2016-ps/provisioning/Provisioning.md [hunk 1]
```diff
@@ -12,30 +12,29 @@ title: Provisioning
 
 # Provisioning Module
 ## Description
-{{Manually Enter Description Here}}
+This reference cmdlet descriptions and syntax for all Provisioning-specific cmdlets. It lists cmdlets in alphabetical order based on the verb at the beginning of the cmdlet.
```
Comment (JohanFreelancer9) [inline docset/winserver2016-ps/provisioning/Provisioning.md:15]:
```suggestion
This reference cmdlet provides descriptions and syntax for all Provisioning-specific cmdlets. It lists cmdlets in alphabetical order based on the verb at the beginning of the cmdlet.
```

File: docset/winserver2016-ps/provisioning/Provisioning.md [hunk 2]
```diff
@@ -12,30 +12,29 @@ title: Provisioning
...
-{{Manually Enter Export-Trace Description Here}}
+Export trace events.
 
 ### [Get-ProvisioningPackage](Get-ProvisioningPackage.md)
-{{Manually Enter Get-ProvisioningPackage Description Here}}
+Gets information about the installed provisioning package.
 
 ### [Get-TrustedProvisioningCertificate](Get-TrustedProvisioningCertificate.md)
-{{Manually Enter Get-TrustedProvisioningCertificate Description Here}}
+List all installed trusted provisioning certificates; use this cmdlet to get the certificate thumbprint to use with the Uninstall-TrustedProvisioningCertificate cmdlet
```
Comment (JohanFreelancer9) [inline docset/winserver2016-ps/provisioning/Provisioning.md:28]:
```suggestion
List all installed trusted provisioning certificates; use this cmdlet to get the certificate thumbprint to use with the Uninstall-TrustedProvisioningCertificate cmdlet.
```

File: docset/winserver2016-ps/provisioning/Provisioning.md [hunk 3]
```diff
@@ -12,30 +12,29 @@ title: Provisioning
...
 
 ### [Install-ProvisioningPackage](Install-ProvisioningPackage.md)
-{{Manually Enter Install-ProvisioningPackage Description Here}}
+Install .PPKG package onto the local machine.
 
 ### [Install-TrustedProvisioningCertificate](Install-TrustedProvisioningCertificate.md)
-{{Manually Enter Install-TrustedProvisioningCertificate Description Here}}
+Adds a certificate to the Trusted Certificate store
```
Comment (JohanFreelancer9) [inline docset/winserver2016-ps/provisioning/Provisioning.md:34]:
```suggestion
Add a certificate to the Trusted Certificate Store.
```

File: docset/winserver2016-ps/provisioning/Provisioning.md [hunk 4]
```diff
@@ -12,30 +12,29 @@ title: Provisioning
...
 
 ### [Uninstall-ProvisioningPackage](Uninstall-ProvisioningPackage.md)
-{{Manually Enter Uninstall-ProvisioningPackage Description Here}}
+Uninstall .PPKG package from the local machine.
 
 ### [Uninstall-TrustedProvisioningCertificate](Uninstall-TrustedProvisioningCertificate.md)
-{{Manually Enter Uninstall-TrustedProvisioningCertificate Description Here}}
-
+Remove a previously installed provisioning certificate
```
Comment (JohanFreelancer9) [inline docset/winserver2016-ps/provisioning/Provisioning.md:40]:
```suggestion
Remove a previously installed provisioning certificate.
```

File: docset/winserver2019-ps/provisioning/Provisioning.md [hunk 5]
```diff
@@ -12,30 +12,29 @@ title: Provisioning
 
 # Provisioning Module
 ## Description
-{{Manually Enter Description Here}}
+This reference cmdlet descriptions and syntax for all Provisioning-specific cmdlets. It lists cmdlets in alphabetical order based on the verb at the beginning of the cmdlet.
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/provisioning/Provisioning.md:15]:
```suggestion
This reference cmdlet provides descriptions and syntax for all provisioning-specific cmdlets. It lists cmdlets in alphabetical order based on the verb at the beginning of the cmdlet.
```

File: docset/winserver2016-ps/provisioning/Provisioning.md [hunk 6]
```diff
@@ -12,30 +12,29 @@ title: Provisioning
 
 # Provisioning Module
 ## Description
-{{Manually Enter Description Here}}
+This reference cmdlet descriptions and syntax for all Provisioning-specific cmdlets. It lists cmdlets in alphabetical order based on the verb at the beginning of the cmdlet.
 
 ## provisioning Cmdlets
 ### [Export-ProvisioningPackage](Export-ProvisioningPackage.md)
-{{Manually Enter Export-ProvisioningPackage Description Here}}
+Extract the contents of a provisioning package.
```
Comment (JohanFreelancer9) [inline docset/winserver2016-ps/provisioning/Provisioning.md:19]:
```suggestion
Extracts the contents of a provisioning package.
```

File: docset/winserver2016-ps/provisioning/Provisioning.md [hunk 7]
```diff
@@ -12,30 +12,29 @@ title: Provisioning
...
 
 ### [Export-Trace](Export-Trace.md)
-{{Manually Enter Export-Trace Description Here}}
+Export trace events.
```
Comment (JohanFreelancer9) [inline docset/winserver2016-ps/provisioning/Provisioning.md:22]:
```suggestion
Exports trace events.
```

File: docset/winserver2016-ps/provisioning/Provisioning.md [hunk 8]
```diff
@@ -12,30 +12,29 @@ title: Provisioning
...
-{{Manually Enter Get-ProvisioningPackage Description Here}}
+Gets information about the installed provisioning package.
 
 ### [Get-TrustedProvisioningCertificate](Get-TrustedProvisioningCertificate.md)
-{{Manually Enter Get-TrustedProvisioningCertificate Description Here}}
+List all installed trusted provisioning certificates; use this cmdlet to get the certificate thumbprint to use with the Uninstall-TrustedProvisioningCertificate cmdlet
 
 ### [Install-ProvisioningPackage](Install-ProvisioningPackage.md)
-{{Manually Enter Install-ProvisioningPackage Description Here}}
+Install .PPKG package onto the local machine.
```
Comment (JohanFreelancer9) [inline docset/winserver2016-ps/provisioning/Provisioning.md:31]:
```suggestion
Installs .PPKG package onto the local machine.
```

File: docset/winserver2016-ps/provisioning/Provisioning.md [hunk 9]
```diff
@@ -12,30 +12,29 @@ title: Provisioning
...
 
 ### [Install-TrustedProvisioningCertificate](Install-TrustedProvisioningCertificate.md)
-{{Manually Enter Install-TrustedProvisioningCertificate Description Here}}
+Adds a certificate to the Trusted Certificate store
 
 ### [Uninstall-ProvisioningPackage](Uninstall-ProvisioningPackage.md)
-{{Manually Enter Uninstall-ProvisioningPackage Description Here}}
+Uninstall .PPKG package from the local machine.
```
Comment (JohanFreelancer9) [inline docset/winserver2016-ps/provisioning/Provisioning.md:37]:
```suggestion
Uninstalls .PPKG package from the local machine.
```

File: docset/winserver2019-ps/provisioning/Provisioning.md [hunk 10]
```diff
@@ -12,30 +12,29 @@ title: Provisioning
...
 
 ## provisioning Cmdlets
 ### [Export-ProvisioningPackage](Export-ProvisioningPackage.md)
-{{Manually Enter Export-ProvisioningPackage Description Here}}
+Extract the contents of a provisioning package.
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/provisioning/Provisioning.md:19]:
```suggestion
Extracts the contents of a provisioning package.
```

File: docset/winserver2019-ps/provisioning/Provisioning.md [hunk 11]
```diff
@@ -12,30 +12,29 @@ title: Provisioning
...
 ### [Export-Trace](Export-Trace.md)
 {{Manually Enter Export-Trace Description Here}}
 
 ### [Get-ProvisioningPackage](Get-ProvisioningPackage.md)
-{{Manually Enter Get-ProvisioningPackage Description Here}}
+Gets information about the installed provisioning package.
 
 ### [Get-TrustedProvisioningCertificate](Get-TrustedProvisioningCertificate.md)
-{{Manually Enter Get-TrustedProvisioningCertificate Description Here}}
+List all installed trusted provisioning certificates; use this cmdlet to get the certificate thumbprint to use with the Uninstall-TrustedProvisioningCertificate cmdlet
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/provisioning/Provisioning.md:28]:
```suggestion
Lists all installed trusted provisioning certificates; use this cmdlet to get the certificate thumbprint to use with the Uninstall-TrustedProvisioningCertificate cmdlet.
```

File: docset/winserver2019-ps/provisioning/Provisioning.md [hunk 12]
```diff
@@ -12,30 +12,29 @@ title: Provisioning
...
 
 ### [Install-ProvisioningPackage](Install-ProvisioningPackage.md)
-{{Manually Enter Install-ProvisioningPackage Description Here}}
+Install .PPKG package onto the local machine.
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/provisioning/Provisioning.md:31]:
```suggestion
Installs .PPKG package onto the local machine.
```

File: docset/winserver2019-ps/provisioning/Provisioning.md [hunk 13]
```diff
@@ -12,30 +12,29 @@ title: Provisioning
...
 
 ### [Install-TrustedProvisioningCertificate](Install-TrustedProvisioningCertificate.md)
-{{Manually Enter Install-TrustedProvisioningCertificate Description Here}}
+Adds a certificate to the Trusted Certificate store
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/provisioning/Provisioning.md:34]:
```suggestion
Adds a certificate to the Trusted Certificate Store.
```

File: docset/winserver2019-ps/provisioning/Provisioning.md [hunk 14]
```diff
@@ -12,30 +12,29 @@ title: Provisioning
...
 
 ### [Uninstall-ProvisioningPackage](Uninstall-ProvisioningPackage.md)
-{{Manually Enter Uninstall-ProvisioningPackage Description Here}}
+Uninstall .PPKG package from the local machine.
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/provisioning/Provisioning.md:37]:
```suggestion
Uninstalls .PPKG package from the local machine.
```

File: docset/winserver2019-ps/provisioning/Provisioning.md [hunk 15]
```diff
@@ -12,30 +12,29 @@ title: Provisioning
...
 
 ### [Uninstall-TrustedProvisioningCertificate](Uninstall-TrustedProvisioningCertificate.md)
-{{Manually Enter Uninstall-TrustedProvisioningCertificate Description Here}}
-
+Remove a previously installed provisioning certificate
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/provisioning/Provisioning.md:40]:
```suggestion
Removes a previously installed provisioning certificate
```

File: docset/winserver2022-ps/provisioning/Provisioning.md [hunk 16]
```diff
@@ -12,30 +12,29 @@ title: Provisioning
 
 # Provisioning Module
 ## Description
-{{Manually Enter Description Here}}
+This reference cmdlet descriptions and syntax for all Provisioning-specific cmdlets. It lists cmdlets in alphabetical order based on the verb at the beginning of the cmdlet.
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/provisioning/Provisioning.md:15]:
```suggestion
This reference cmdlet provides descriptions and syntax for all provisioning-specific cmdlets. It lists cmdlets in alphabetical order based on the verb at the beginning of the cmdlet.
```

File: docset/winserver2022-ps/provisioning/Provisioning.md [hunk 17]
```diff
@@ -12,30 +12,29 @@ title: Provisioning
...
 
 ## provisioning Cmdlets
 ### [Export-ProvisioningPackage](Export-ProvisioningPackage.md)
-{{Manually Enter Export-ProvisioningPackage Description Here}}
+Extract the contents of a provisioning package.
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/provisioning/Provisioning.md:19]:
```suggestion
Extracts the contents of a provisioning package.
```

File: docset/winserver2022-ps/provisioning/Provisioning.md [hunk 18]
```diff
@@ -12,30 +12,29 @@ title: Provisioning
...
 
 ### [Export-Trace](Export-Trace.md)
-{{Manually Enter Export-Trace Description Here}}
+Export trace events.
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/provisioning/Provisioning.md:22]:
```suggestion
Exports trace events.
```

File: docset/winserver2022-ps/provisioning/Provisioning.md [hunk 19]
```diff
@@ -12,30 +12,29 @@ title: Provisioning
...
 
 ### [Get-ProvisioningPackage](Get-ProvisioningPackage.md)
-{{Manually Enter Get-ProvisioningPackage Description Here}}
+Gets information about the installed provisioning package.
 
 ### [Get-TrustedProvisioningCertificate](Get-TrustedProvisioningCertificate.md)
-{{Manually Enter Get-TrustedProvisioningCertificate Description Here}}
+List all installed trusted provisioning certificates; use this cmdlet to get the certificate thumbprint to use with the Uninstall-TrustedProvisioningCertificate cmdlet
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/provisioning/Provisioning.md:28]:
```suggestion
Lists all installed trusted provisioning certificates; use this cmdlet to get the certificate thumbprint to use with the Uninstall-TrustedProvisioningCertificate cmdlet.
```

File: docset/winserver2022-ps/provisioning/Provisioning.md [hunk 20]
```diff
@@ -12,30 +12,29 @@ title: Provisioning
...
 
 ### [Install-ProvisioningPackage](Install-ProvisioningPackage.md)
-{{Manually Enter Install-ProvisioningPackage Description Here}}
+Install .PPKG package onto the local machine.
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/provisioning/Provisioning.md:31]:
```suggestion
Installs .PPKG package onto the local machine.
```

File: docset/winserver2022-ps/provisioning/Provisioning.md [hunk 21]
```diff
@@ -12,30 +12,29 @@ title: Provisioning
...
 
 ### [Install-TrustedProvisioningCertificate](Install-TrustedProvisioningCertificate.md)
-{{Manually Enter Install-TrustedProvisioningCertificate Description Here}}
+Adds a certificate to the Trusted Certificate store
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/provisioning/Provisioning.md:34]:
```suggestion
Adds a certificate to the Trusted Certificate Store.
```

File: docset/winserver2022-ps/provisioning/Provisioning.md [hunk 22]
```diff
@@ -12,30 +12,29 @@ title: Provisioning
...
 
 ### [Uninstall-ProvisioningPackage](Uninstall-ProvisioningPackage.md)
-{{Manually Enter Uninstall-ProvisioningPackage Description Here}}
+Uninstall .PPKG package from the local machine.
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/provisioning/Provisioning.md:37]:
```suggestion
Uninstalls .PPKG package from the local machine.
```

File: docset/winserver2022-ps/provisioning/Provisioning.md [hunk 23]
```diff
@@ -12,30 +12,29 @@ title: Provisioning
...
 
 ### [Uninstall-TrustedProvisioningCertificate](Uninstall-TrustedProvisioningCertificate.md)
-{{Manually Enter Uninstall-TrustedProvisioningCertificate Description Here}}
-
+Remove a previously installed provisioning certificate
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/provisioning/Provisioning.md:40]:
```suggestion
Removes a previously-installed provisioning certificate
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: docset/winserver2016-ps/provisioning/Provisioning.md [hunk 24]
```diff
@@ -12,30 +12,29 @@ title: Provisioning
...
+Adds a certificate to the Trusted Certificate Store.
 
 ### [Uninstall-ProvisioningPackage](Uninstall-ProvisioningPackage.md)
-{{Manually Enter Uninstall-ProvisioningPackage Description Here}}
+Uninstalls .PPKG package from the local machine.
 
 ### [Uninstall-TrustedProvisioningCertificate](Uninstall-TrustedProvisioningCertificate.md)
-{{Manually Enter Uninstall-TrustedProvisioningCertificate Description Here}}
-
+Removes a previously-installed provisioning certificate.
```
Comment (JohanFreelancer9) [inline docset/winserver2016-ps/provisioning/Provisioning.md:40]:
```suggestion
Removes a previously installed provisioning certificate.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies @velkovb, my bad. One more, please.

File: docset/winserver2016-ps/provisioning/Provisioning.md [hunk 25]
```diff
@@ -12,30 +12,29 @@ title: Provisioning
 
 # Provisioning Module
 ## Description
-{{Manually Enter Description Here}}
+This reference cmdlet provides descriptions and syntax for all provisioning-specific cmdlets. It lists cmdlets in alphabetical order based on the verb at the beginning of the cmdlet.
```
Comment (dariomws) [inline docset/winserver2016-ps/provisioning/Provisioning.md:15]:
```suggestion
This reference provides descriptions and syntax for all provisioning-specific cmdlets. It lists cmdlets in alphabetical order based on the verb at the beginning of the cmdlet.
```

**Review by dariomws — COMMENTED** [review]
We can remove the word cmdlet here.

**Review by dariomws — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Comment by v-dihans** [comment]
Hi @velkovb 

**Following are blocking issues that need to be resolved before we can merge:**

All Articles in the PR
- Fix the syntax. Some of the changes result in {{Fill in the Synopsis}} displaying on staging. 

***

**Reference**

[Criteria used for pull request reviews](https://review.docs.microsoft.com/en-us/office-authoring-guide/pr-criteria?branch=master)

**Blocking**
- The content preview must be clean on staging. It cannot contain any obvious formatting issues.

***

Thanks

**Comment by velkovb** [comment]
@v-dihans @e0i I don't think I understand what the issues is. This PR actually removes all of the {{Fill in the Synopsis}} occurrences. 

Where I can see those syntax errors?

**Comment by v-dihans** [comment]
@velkovb   

You can see staging for each file by clicking on "view" in the Validation report in the PR. 

Thanks

**Comment by velkovb** [comment]
@v-dihans I don't have access to open those.

**Comment by v-dihans** [comment]
@velkovb  

I'll try to help. I'll leave a comment with a screenshot of the staged content for each file.

docset/winserver2016-ps/provisioning/Provisioning.md
[
image
](url)

**Comment by v-dihans** [comment]
docset/winserver2019-ps/provisioning/Provisioning.md

[
image
](url)

**Comment by v-dihans** [comment]
docset/winserver2022-ps/provisioning/Provisioning.md

image

**Comment by velkovb** [comment]
That doesn't seem right. This PR is fixing exactly those issues. Not sure why it is not picking up the change properly.

**Comment by velkovb** [comment]
I guess this is automatically picking up the Synopsis inside the actual cmdlet articles.

**Comment by e0i** [comment]
@sdwheeler It seems that the synopsis fields that we have provided manually are overridden by the build system with the `{{Fill in the Synopsis}}` fields. Can you take a look as the time permits? Thank you.

**Comment by sdwheeler** [comment]
@velkovb @e0i The cmdlets files are missing the Synopsis information. Those need to be updated as well.

**Comment by sdwheeler** [comment]
All of these files need to be updated. All of them are missing Synopsis and other descriptions.

- Export-Trace.md
- Get-TrustedProvisioningCertificate.md
- Install-TrustedProvisioningCertificate.md
- Uninstall-ProvisioningPackage.md
- Uninstall-TrustedProvisioningCertificate.md
-

**Comment by e0i** [comment]
@tiburd This PR is finally green and can be merged. Thanks.

**Pull request merged**